runtime/compiler/x/env/J9CPU.cpp

```diff
@@ -136,5 +131,139 @@ CPU::getX86ProcessorFeatureFlags8()
    return self()->queryX86TargetCPUID()->_featureFlags8;
    }
 
-}
-}
+bool
+J9::X86::CPU::is_test(OMRProcessorArchitecture p)
+   {
+#if !defined(J9VM_OPT_JITSERVER)
```

Comment:
Linux builds have `J9VM_OPT_JITSERVER` defined. Isn't this essentially disabling the test for Linux?

Reply:
It's disabling the test for Linux JITServer. NonJITServer on Linux still goes through with the test.

Reply:
actually I know what you mean now. Yeah I think I need to use the isOutOfProcessCompilation() test instead.